addons/arsenal/functions/fnc_onArsenalClose.sqf

```diff
@@ -69,6 +69,7 @@ camDestroy GVAR(camera);
 if (!isNil QGVAR(moduleUsed)) then {
     GVAR(moduleUsed) = nil;
     objNull remoteControl GVAR(center);
+    player switchCamera cameraView;
```

Comment:
`player` or `ACE_player` ? How does this relate to L62?

Reply:
L62 is only in 3den and when not using Zeus, while the bug happens in any form of Zeus such as on dedicated servers.